src/main/java/io/vinyldns/java/model/batch/CreateBatchRequest.java

```diff
@@ -64,4 +66,14 @@ public int hashCode() {
 
     return Objects.hash(comments, changes);
   }
+  
+  public String getOwnerGroupId() {
+      return ownerGroupId;
+  }
+
+  public void setOwnerGroupId(String ownerGroupId) {
+      if (ownerGroupId != null) {
```

Comment:
We shouldn't set a condition on this because `null` is a valid value here (indicating that `ownerGroupId` is not provided).